packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -332,6 +332,10 @@ export default class QuanticCategoryFacet extends LightningElement {
     return this.withSearch && !!this.input?.value?.length;
   }
 
+  get categoryFacetLiClass() {
+    return this.activeParent ? 'slds-var-m-left_large slds-grid' : 'slds-grid';
```

Comment:
```suggestion
    return `slds-grid ${this.activeParent ? 'slds-var-m-left_large': ''}`;
```

Personal choice if you want to change it but I prefer it like this.